.gitignore

```diff
@@ -35,3 +35,8 @@ coverage
 
 /.vscode/*
 /.devcontainer/*
+
+e2e/videos
+e2e/screenshots
+e2e/browsers
+e2e/config/local.json
```

Comment:
Can we use a name that's a little bit more descriptive? `e2e-tests` perhaps?

Also, missing a newline here.

Reply:
https://github.com/vscode-icons/vscode-icons/wiki/ListOfFolders#e2e

Reply:
team voted `end-to-end-tests`